vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -5544,6 +5544,21 @@ unit:EuropeanUnitOfAccount9
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "European Unit of Account 9 (E.U.A.-9) (Bonds market unit)"@en ;
 .
+unit:ExaBYTE
+  a qudt:Unit ;
+  dcterms:description "The exabyte is a multiple of the unit byte for digital information. The prefix exa means 10^18 in the International System of Units (SI), however, in information quantities kilo=1024 so ExaByte is 1024^6 Bytes."^^rdf:HTML ;
```

Comment:
This description needs to be adjusted to state that ExaBYTE is 10^18 Bytes and not 1024^6 Bytes.